app/browser/tabs.js

```diff
@@ -721,6 +721,16 @@ const api = {
         }
       })
 
+      tab.on('did-detach', (e, oldTabId) => {
```

Comment:
@Slava is this block supposed to be here?

Reply:
I don't understand why this is needed. Maybe you can remember the context of this commit? https://github.com/brave/browser-laptop/pull/14734/commits/c375ba35c4192da94e2c3dc1aa9f9958a9ce1044

Reply:
@Slava lol wow i have no idea. sorry i thought it was left over from the rebase.

Reply:
er yeah this needs to - looks like a rebase issue from before -> after the signle-webview project. Tab's now `detach` every time they are made inactive / active, so this will break something (minor).

Reply:
@petemill sorry do you mean this chunk should be deleted or kept (or other)?

Reply:
@petemill ping

Reply:
looks like this block got added during rebase because of a fix within in the original ethwallet branch - https://github.com/brave/browser-laptop/commit/00cdfdd884cf4ba23e141b0693cb7ae940b80ce1#diff-2be38b3d97f35cd818780bac4be979c4R629

I think we should delete this block

Reply:
oops missed out a key word

> this needs to **go**

Thanks for the spot